tests/test_suite/start.py

```diff
@@ -300,7 +300,8 @@ def lift_test_cfg(toolset, test):
 
   command_line = [toolset["mcsema-lift"], "--arch", test.architecture(),
                   "--os", test.platform(), "--cfg", test.cfg_path(),
-                  "--output", output_file_path]
+                  "--output", output_file_path, "--libc_constructor", "init",
+                  "--libc_destructor", "fini"]
 
```

Comment:
Can you add brief explanations of these two new command line arguments ("--libc_constructor", "--libc_destructor") to docs/CommandLineReference.md and add that commit to this pull request?